1469_Find_All_The_Lonely_Nodes/try_1.cpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * Definition for a binary tree node.
+ * struct TreeNode {
+ *     int val;
+ *     TreeNode *left;
+ *     TreeNode *right;
+ *     TreeNode() : val(0), left(nullptr), right(nullptr) {}
+ *     TreeNode(int x) : val(x), left(nullptr), right(nullptr) {}
+ *     TreeNode(int x, TreeNode *left, TreeNode *right) : val(x), left(left), right(right) {}
+ * };
+ */
+class Solution {
+private:
+    void helper(TreeNode* root, vector<int>& ans) {
+        if (!root) return;

```

Comment:
```suggestion
        if (root==nullptr) return;
```
<僅提供參考> 這樣可以明確表明root是一個指標(可讀性較高)，`(!root)`可以用是因為c++會幫忙轉型成bool，用`nullptr`會是一個安全且可讀性高的寫法(雖然我都偷懶寫`(!root)`)